velox/exec/Spill.cpp

```diff
@@ -399,6 +399,45 @@ void FileSpillMergeStream::close() {
   spillFile_.reset();
 }
 
+std::unique_ptr<SpillMergeStream> ConcatenateFilesSpillMergeStream::create(
+    uint32_t id,
+    std::vector<std::unique_ptr<SpillReadFile>> spillFiles) {
+  auto spillStream = std::unique_ptr<ConcatenateFilesSpillMergeStream>(
+      new ConcatenateFilesSpillMergeStream(id, std::move(spillFiles)));
+  spillStream->nextBatch();
+  return spillStream;
+}
+
+uint32_t ConcatenateFilesSpillMergeStream::id() const {
+  return id_;
+}
+
+void ConcatenateFilesSpillMergeStream::nextBatch() {
+  VELOX_CHECK(!closed_);
+  index_ = 0;
+  for (; fileIndex_ < spillFiles_.size(); ++fileIndex_) {
+    if (spillFiles_[fileIndex_]->nextBatch(rowVector_)) {
+      VELOX_CHECK_NOT_NULL(rowVector_);
+      size_ = rowVector_->size();
+      return;
+    }
```

Comment:
Do we want to reset the finished spill file reader to release memory earlier? Thanks!